src/main/java/org/ibp/api/java/impl/middleware/dataset/DatasetServiceImpl.java

```diff
@@ -295,6 +295,7 @@ public List<DatasetDTO> getDatasets(final Integer studyId, final Set<Integer> da
 		final List<DatasetDTO> datasetDTOs = new ArrayList<>();
 		for (final org.generationcp.middleware.domain.dms.DatasetDTO datasetDTO : datasetDTOS) {
 			final DatasetDTO datasetDto = mapper.map(datasetDTO, DatasetDTO.class);
+			// TODO: should also rename environment dataset?
```

Comment:
@lik-83 anything we should do with this TODO?

Reply:
I removed this TODO.